### DISCUSSION TITLE
Add undo button to the popup to restore deleted item

### DISCUSSION THREAD

**Discussion by lucius346346** [karakeep-app/karakeep#558] [Ideas]
We already have a popup that informs users when an item is deleted. How about enhancing it by adding an 'Undo' button? This would be particularly useful since Hoarder lacks a recycle bin functionality.
![image](https://github.com/user-attachments/assets/a617c076-1af9-41cf-b6d5-41e8b1ea75a1)

**Comment by kamtschatka** [comment]
Please make sure to use the search. There is already a feature request: https://github.com/hoarder-app/hoarder/issues/39
